jujugui/static/gui/src/app/components/mid-point/test-mid-point.js

```diff
@@ -111,7 +111,8 @@ describe('MidPoint', function() {
           <ul className="mid-point__tag-list">
             {juju.components.MidPoint.prototype.tags}
           </ul>
-          <button className="mid-point__store-button">
+          <button className="mid-point__store-button"
+            onClick={output.props.children[2].props.children[1].props.onClick}>
```

Comment:
Is is possible to target a class here instead of an index as this test will fail if the order of buttons change?